### DISCUSSION TITLE
Export to Markdown?

### DISCUSSION THREAD

**Discussion by mattkang** [jupyter-book/mystmd#1697] [Q&A]
Is it possible to export Myst markdown to vanilla markdown, e.g. GFM, CommonMark?

I want to be using Myst more but need to preserve compatibility. Tried to run `myst build --md` and the export retained things like header targets, i.e. `(my-section)=`. Would expect features like cross-referencing to get hard-coded as they do when exporting to other formats.

**Comment by riziles** [comment]
Jupytext is your best bet
